Wordle/Wordle/FileIO.cpp

```diff
@@ -0,0 +1,79 @@
+#include "FileIO.h"
+
+#include <iostream>
+#include <string>
+#include <fstream>
+
+void readListWords(const char* fileName, std::string* array, unsigned size)
+{
+	std::fstream file;
+	file.open(fileName, std::ios::in);
+
+	if(!file)
+	{
+		std::cout << "ERROR READING FILE\n";
+		//TODO ERROR
+	}
+	else
+	{
+		for (unsigned i = 0; i < size && !file.eof(); i++)
+		{
+			file >> array[i];
+		}
+		file.close();
+	}
+}
+
+void readFile(const char* fileName, int& gamesPlayed, int& gamesWon, int& gamesLost, int& points, std::string& dateTime)
```

Comment:
read what file? I suppose it some kind of stats, so maybe readFileStats or readStatsFile or similar